component/component.go

```diff
@@ -60,6 +61,16 @@ type Component interface {
 	Shutdown(ctx context.Context) error
 }
 
+// Settings is passed to ReceiverFactory.Create* functions.
+type Settings struct {
```

Comment:
Is this used anywhere?

Reply:
This will replace `ExporterCreateParams`, `ReceiverCreateParams`, `ExtensionCreateParams` and `ProcessorCreateParams` to standarize this. I've placed it here to make it easier to refactor Receivers, Processor, Extensions and Exporters.

Reply:
I think for the moment we should remove it since is not used, and add it in the next PR :)